### ISSUE TITLE
Calculate optimal hyperparameters in nested cross-validation

### ISSUE THREAD

**Issue by munterkalmsteiner** [bth-dipt-research/SVAR#1]
Currently, we do log all best hyperparameters and evaluation results, for each outer fold. To choose the best hyperparameters for final model training, we should calculate the best hyperparameters as follows:

- mean of learning rate
- round(mean of epochs)
- mode of batch size
- round(mean of seed)
- round(mean of max iter)
- most frequent of solver

The final hyperparameters should be printed out at the end of the log.

**Comment by munterkalmsteiner** [comment]
Fixed  in b0bdba49e1f5ef74472107982b928b9b1499daab
